common/Util-desktop.cpp

```diff
@@ -160,6 +162,56 @@ std::size_t getTotalSystemMemoryKb()
     return totalMemKb;
 }
 
+size_t getMaxConcurrentTCPConnections(const size_t minimum)
+{
+#ifdef __linux__
+    auto readDecimal = [](const char* path, const size_t defaultValue) -> size_t {
+        const int fd = ::open(path, O_RDONLY);
+        if (fd<0)
```

Comment:
inconsistent whitespace in your own code here; "if (fd < 0)" surely ? =)

Reply:
Will use FileUtil::readFile then, closes thing we have avoiding fopen/fgets.

Reply:
OK, I remember, FileUtil::read is not available to all link targets. So it is fgets/fopen then.